src/routes/(console)/project-[region]-[project]/databases/database-[database]/collection-[collection]/+page.svelte

```diff
@@ -88,14 +87,6 @@
             $isCsvImportInProgress = false;
         }
     }
-
-    /**
-     * Controls visibility of CSV Imports feature:
-     * - Shown if running on self-hosted
-     * - Shown on cloud only if the organization is Appwrite's.
-     * - Hidden on cloud for any non-Appwrite organization.
-     */
-    $: showCsvImports = !isCloud || $organization.$id === APPWRITE_OFFICIALS_ORG;
```

Comment:
how is the `!isCloud` setting handled?

Reply:
Good to leave out . We can enable all on self-hosted, to combat this